paasta_tools/autoscaling_lib.py

```diff
@@ -664,23 +649,38 @@ def wait_and_terminate(slave, drain_timeout, dry_run, region=None):
                 raise
 
 
+def sort_slaves_to_kill(slaves):
+    """Pick the best slaves to kill. This returns a list of slaves
+    after sorting in preference of which slaves we kill first.
+    It sorts first by number of chronos tasks, then by total number of tasks
+
+    :param slaves: list of slaves dict
+    :returns: list of slaves dicts"""
+    if slaves:
```

Comment:
if slaves is a list, then you can just return the list comprehension (because the comprehension over an empty list will return an empty list)